src/argilla/server/schemas/v1/responses.py

```diff
@@ -18,12 +18,15 @@
 
 from pydantic import BaseModel
 
+from argilla.server.models import ResponseStatus
+
 
 class Response(BaseModel):
     id: UUID
     values: Dict[str, Any]
     record_id: UUID
     user_id: UUID
+    status: ResponseStatus
```

Comment:
In `src/argilla/server/schemas/v1/datasets.py`, there exists a distinct `Response` schema. Would you suggest including the `status` field in this schema as well?

Reply:
Oh, I didn't realize this `Response` schema. Yes, we should include the `status` field here too, otherwise when calling the listing records `/api/v1/datasets/{dataset_id}/records` the status for each response won't be there, and the frontend would have to send an additional request to get the status for each record, which is not ideal.

I think it makes sense to have a `ResponseBase` class in `src/argilla/server/schemas/v1/responses.py`, this way we avoid code duplication. 

- We can create a `Response`class inheriting from `ResponseBase` class with additional attributes that have to be exposed when listing responses (like the `record_id`) in `src/argilla/server/schemas/v1/datasets.py`.
- We can create a `Response` class inheriting from `ResponseBase` with no additional attributes that can be used when listing records in `src/argilla/server/schemas/v1/datasets.py`.